media/js/privacy/privacy-protocol.js

```diff
@@ -19,3 +19,16 @@
         buttons[i].setAttribute('data-close', closeText);
     }
 })();
+
+if (window.location.href.includes('cookies')) {
+    window.onload = function() {
```

Comment:
There should be no need to override the `onload` event handler here? Suggest moving this logic inside the closure above, together with the existing code for this page.

Reply:
You are right. Thought it was necessary but defining the variables after the existing code works also.